components/centraldashboard/public/components/dashboard-view.pug

```diff
@@ -1,25 +1,26 @@
-paper-card#Getting-Started(heading='Getting Started')
-    template(is='dom-repeat', items='[[gettingStartedItems]]')
-        a.heading(href$='[[item.link]]', tabindex='-1',
-                    target='_blank')
-            paper-icon-item
-                iron-icon(icon='launch', slot='item-icon')
-                paper-item-body(two-line)
-                    .header [[item.text]]
-                    aside(secondary) [[item.desc]]
-paper-card.thin#Quick-Links(heading='Quick shortcuts')
-    template(is='dom-repeat', items='[[quickLinks]]')
-        iframe-link.link(href$="[[item.link]]", tabindex='-1')
-            paper-icon-item [[item.text]]
-                paper-icon-button.button(icon='arrow-forward',
-                    slot='item-icon', alt='[[item.text]]')
+.column
+    paper-card#Quick-Links(heading='Quick shortcuts')
+        template(is='dom-repeat', items='[[quickLinks]]')
+            iframe-link.link(href$="[[item.link]]", tabindex='-1')
+                paper-icon-item [[item.text]]
+                    paper-icon-button.button(icon='arrow-forward',
+                        slot='item-icon', alt='[[item.text]]')
+.column
+    paper-card#Getting-Started(heading='Getting Started')
+        template(is='dom-repeat', items='[[gettingStartedItems]]')
+            a.heading(href$='[[item.link]]', tabindex='-1',
+                        target='_blank')
+                paper-icon-item
+                    iron-icon(icon='launch', slot='item-icon')
+                    paper-item-body(two-line)
+                        .header [[item.text]]
+                        aside(secondary) [[item.desc]]
+.column
     template(is='dom-if', if='[[platformLinks]]')
-        section#Platform-Links
-            header [[platformName]] Information
+        paper-card#Platform-Links(class$='[[platformName]]', heading='[[platformTitle]]', image='[[platformLogo]]')
```

Comment:
It looks like this DOM structural change breaks the unit test [here](https://github.com/kubeflow/kubeflow/blob/0d85f7c7a2e22abffad66849a77b6d7242dcd36f/components/centraldashboard/public/components/dashboard-view_test.js#L49). Presubmit tests pass because we don't do client-side tests yet in the presubmit environment because there's a bunch of roundabout stuff necessary to get headless Chrome to work. 

For now, I've just been issuing `npm run coverage` before pushing to make sure everything passes and coverage is high.

Reply:
![image](https://user-images.githubusercontent.com/5303018/57564655-2e2aee00-7364-11e9-96af-df4aa9d774f5.png)

Fixed